Add files via upload

AndrewLucia opened the pull request:
Beginning work on two backend tickets Backend: Login page with https and Backend: Checkin API calls and sessions. Not all API calls done.

AndrewLucia committed:
Add files via upload

LukeBoyer commented on c4c-internal-backend-master/api/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:
```diff
@@ -0,0 +1,5 @@
+C:\Users\Andrew Lucia\Documents\C4C\c4c-internal-backend-master\api\src\main\java\com\codeforcommunity\dto\MemberReturn.java
```
please remove this file and as well as extraneous binaries and jars. Artifacts and as well as machine specific files should not be shared on git with src code.

LukeBoyer commented on c4c-internal-backend-master/service/src/main/java/com/codeforcommunity/ServiceMain.java:
```diff
@@ -0,0 +1,60 @@
+   * Connect to the database and create a DSLContext so jOOQ can interact with it.
+   */
+  private void connectDb() {
+    //This block ensures that the MySQL driver is loaded in the classpath
```
mysql or postgres?

LukeBoyer commented on c4c-internal-backend-master/service/src/main/java/com/codeforcommunity/ServiceMain.java:
```diff
@@ -0,0 +1,60 @@
+    try {
+      Class.forName("org.postgresql.Driver");
+    } catch (ClassNotFoundException e) {
+      e.printStackTrace();
```
consider exposing this or another exception instead of continuing in this method here if class is not found.

LukeBoyer commented on c4c-internal-backend-master/service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java:
```diff
@@ -0,0 +1,56 @@
+        + "   from member\n"
+        + "   where first_name = ? and last_name = ?;", first, last);
+
+    if (result.isEmpty())
```
style: use brackets around this statement

LukeBoyer commented on c4c-internal-backend-master/api/src/main/java/com/codeforcommunity/README.md:
```diff
@@ -0,0 +1 @@
+
```
Consider writing a readme or removing this blank file

LukeBoyer reviewed (changes requested):
Please address comments and reach out if you have any questions. Great work!

Also it is imperative that repositories are kept clean, be careful with adding and committing every file as there are a lot of binaries that we don't want. In addition we don't want to be committing artifact files

LukeBoyer commented on c4c-internal-backend-master/persist/src/main/resources/db/migration/V1__Initial_Import.sql:
```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS member (
+  id VARCHAR(36) NOT NULL PRIMARY KEY,
```
not null is implied with the primary key. Also look into the serial keyword.

VishikBhalla committed:
Added HTTPS to Login Page

User now has to enter https://localhost:8443/login to get to login page, where they can enter their username and password with a query string.

AndrewLucia committed:
Merge branch 'master' into Backend-Login-page-with-https

AndrewLucia committed:
removing ignored files
